.travis.yml

```diff
@@ -78,7 +78,21 @@ jobs:
           kubernetes.yml \
           openssl-self-signed-certificate.yml \
         || exit 1
+    - name: "hadolint"
+      language: shell
+      script: |
+        find . -name "Dockerfile" -type f -print0 \
+        | xargs -0 -I file sh -c 'docker run --rm -i hadolint/hadolint:v1.17.5-8-gc8bf307-alpine < "file"' \
+        || exit 1
+    - name: "Build kitesproject/net-tests container image"
+      stage: build_container_images
+      language: shell
+      before_script: cd docker/kites/net-tests || exit 1
+      script: |
+        docker build --target=final -t kitesproject/net-test . \
+        && docker build --target=dev -t kitesproject/net-test .
```

Comment:
Use two different steps, so we can test the images independently

Reply:
Ok!